README.md

```diff
@@ -163,7 +170,12 @@ If you see any missing data please [contact us](#provide-feedback).
 
 ## Multi Platform Support
 
-PcapPlusPlus is currently supported on __Windows__, __Linux__, __MacOS__, __Android__ and __FreeBSD__. Please visit PcapPlusPlus web-site to see all of the [supported platforms](https://pcapplusplus.github.io/docs/platforms) and refer to the [Download](#download) section to start using PcapPlusPlus on your platform.
+PcapPlusPlus is currently supported on
+__Windows__ <picture><img src="https://github.com/PcapPlusPlus/pcapplusplus.github.io/raw/master/static/img/os-logos/logo-windows-small.png" alt=""/></picture>,
+__Linux__ <picture><img src="https://github.com/PcapPlusPlus/pcapplusplus.github.io/raw/master/static/img/os-logos/logo-linux-small.png" alt=""/></picture>,
+__MacOS__ <picture><img src="https://github.com/PcapPlusPlus/pcapplusplus.github.io/raw/master/static/img/os-logos/logo-apple-dark-small.png" alt=""/></picture>,

```

Comment:
I think we need to modify the Apple icon to be on a black background otherwise, it doesn't show in light mode:
![image](https://user-images.githubusercontent.com/9059541/194226848-7e5a72d7-787b-45b1-9ada-883230204190.png)

Reply:
@seladb Done!

Reply:
Nice idea using the `<picture>` element!!

Few improvements we can do since we're using the `<picture>` and `<img>` elements:
- For MacOS we don't need 2 `<source>` elements, one is enough:
  ```html
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://github.com/PcapPlusPlus/pcapplusplus.github.io/raw/master/static/img/os-logos/logo-apple-dark-small.png"/>
    <img alt="" src="https://github.com/PcapPlusPlus/pcapplusplus.github.io/raw/master/static/img/os-logos/logo-apple-small.png"/>
  </picture>
  ```
- For all images - since we're using `<img>` we can go back to use the standard logos (non-small) and add `width=16` and `height=16` to each image:
  ```html
  <picture>
    <img src="https://github.com/PcapPlusPlus/pcapplusplus.github.io/raw/master/static/img/os-logos/logo-windows.png" alt="" width="16" height="16"/>
  </picture>
  ```